.github/workflows/publish.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          platforms: linux/amd64,linux/arm64
```

Comment:
Is this correct?

Reply:
You mean the format? It should be, see [Customizing](https://github.com/docker/build-push-action#customizing) chapter of `build-push-action`.  [Here](https://docs.docker.com/build/ci/github-actions/examples/#push-to-multi-registries) is the same configuration in official Docker documentation.

Reply:
I am not sure I understand that notation, but it seems to be correct :)